src/fixes.rs

```diff
@@ -17,7 +18,10 @@ trait Fix {
     ) -> Option<usize> {
         let mut count: usize = 0;
         for warning in warnings {
-            let line = lines.get_mut(warning.line_number() - 1)?;
+            let line = lines
+                .get_mut(warning.line_number() - 1)
+                .expect("warning references inexistent line");
```

Comment:
I guess that this should be hard error, instead of silent pass?

Also, I'm feeling slightly uncomfortable about relating on the fact that the lines count/order is preserved after previous fixers... For example, fixer for UnorderedKey can mangle the lines order here. Wouldn't it be better for Warning to store &mut reference to the line instead of it's clone? Or maybe we should use `find` based on `line_number` here instead of `get_mut`?